server/py/services/api/crud/alert_activation.py

```diff
@@ -77,6 +80,7 @@ def _prepare_notifications_states(
         for kind, status_info in notification_errors.items():
             errors = list(status_info["errors"])
             success_count = status_info["success_count"]
+            failed_count = status_info["failed_count"]
```

Comment:
I'd do a `get` to be a bit more defensive
```suggestion
            success_count = status_info.get("success_count", 0)
            failed_count = status_info.get("failed_count", 0)
```